python/doc/theory/data_analysis/quantile_asymptotic_confidence_estimation.rst

```diff
@@ -0,0 +1,100 @@
+.. _quantile_asymptotic_confidence_estimation:
+
+Asymptotic quantile confidence interval based on order statistics
+-----------------------------------------------------------------
+
+We consider a random variable :math:`X` of dimension 1 and its quantile :math:`x_{\alpha}`
+of level :math:`\alpha` (:math:`\alpha \in [0, 1]`).
```

Comment:
<img width="1263" height="220" alt="image" src="https://github.com/user-attachments/assets/da4dee50-1ad7-403d-9b8b-12b651d6c83b" />

This is smoother:

```suggestion
of level :math:`\alpha` (where :math:`\alpha \in [0, 1]`).
```